### PR TITLE
upgrade d3 to latest

### PR DESCRIPTION
**Pull request by jvswatson**

### CODE REVIEW

**Comment by jasonjohnson115** [comment]
+1

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "author": "Dataminr",
   "name": "dataminr-react-components",
   "description": "Collection of reusable React Components and utility functions",
-  "version": "0.8.1",
+  "version": "0.9.0",
```
Comment (ernieturner) [inline package.json]: Is there anyway for this to be a breaking change? I'm not sure if there's a reason we couldn't just go to 0.8.2.

Comment (jasonjohnson115) [inline package.json]: The only potential side effect is 2 copies of d3 being used side by side. I think Utah is the only repo we have that does this. If anyone else is using the repo, I'd say it's on them to fix that and nothing is actually broken.

Comment (jvswatson) [inline package.json]: I wasn't sure if we would be concerned about consumers potentially now using two versions of d3 (4.2.6 with drc and 3.5.12 with their own code).  Certainly our own interface isn't changing, but I wasn't sure if the dependency changes didn't warrant a minor bump.

Comment (jvswatson) [inline package.json]: okay, I'll just bump it to 0.8.2

**Pull request merged**